api/api.proto

```diff
@@ -4819,6 +4828,25 @@ message SdkAutoFSTrimStatusResponse {
   string message = 2;
 }
 
+// SdkAutoFSTrimUsageRequest defines a request to get status of autofs trim operation
+message SdkAutoFSTrimUsageRequest {
+}
+
+// SdkAutoFSTrimUsageResponse defines the response for a
+// SdkAutoFSTrimUsageRequest.
+message SdkAutoFSTrimUsageResponse {
+  // map of volume id and volume name
+  map<string, string> vol_name = 1;
+  // map of volume id and the volume size
+  map<string, uint64> vol_size = 2;
+  // map of volume id and the du usage
+  map<string, uint64> du_usage = 3;
+  // map of volume id and the px usage
+  map<string, uint64> px_usage = 4;
+  // Text blob containing ASCII text providing details of the operation
+  string message = 5;
```

Comment:
can we have a struct instead of 4 different maps
like 
map<string, fstrimVolUsageInfo>